test/cpp/strings.unit.cpp

```diff
@@ -0,0 +1,128 @@
+#define MICROSTRAIN_HAS_STD_STRING 1
+
+#include <string_view>
+
+#include <microstrain_test.hpp>
+#include <microstrain/strings.hpp>
+
+static constexpr char CHECK_STRING[] = "Test: \"quotes\" 'single' & <xml/> {json} [array] $100 €50 ¥200 @user #tag 🚀 \n\t\\ 100% café naïve";
+size_t CHECK_STRING_LENGTH = 102; // Length without null terminator
+
+struct BufferWrapper
+{
+    char array[1024] = {};
+    microstrain::Span<char> span{array};
```

Comment:
Don't span a member, it should be a function returning a span. It's a view of array, not a data value of its own.